WordPress/gutenberg
# [RNMobile] react native bridge UBE path fix

Gutenberg Mobile PR https://github.com/wordpress-mobile/gutenberg-mobile/pull/3353


## Description

During the composite build project, changes made here https://github.com/WordPress/gutenberg/pull/28216 caused the UBE path to be broken so this PR seeks to address this by including the additional `../` needed in the path to resolve the UBE assets needed by the editor. 

This PR also builds on top of the broken symlink that links the assets in `react-native-bridge`. Those changes were merged in here https://github.com/WordPress/gutenberg/pull/30388
The symlink was updated in this PR as well, as it was still broken even though the diff changes look near similar. 

## How has this been tested?
1. Open the editor. 
2. Edit an unsupported block. 
3. Ensure there is no crash. 

## Types of changes
The `assets.srcDirs` path pointing to the `resources/unsupported-block-editor` folder with `gutenberg-mobile` was updated. 





## Checklist:
